portality/crosswalks/article_form.py

```diff
@@ -1,41 +1,53 @@
 from portality import models
+from portality.view.forms import AuthorForm
+
 
 class ArticleFormXWalk(object):
     format_name = "form"
 
-    def crosswalk_form(self, form, add_journal_info=True, limit_to_owner=None):
-        article = models.Article()
-        bibjson = article.bibjson()
+    def crosswalk_form(self, form, add_journal_info=True, limit_to_owner=None, id=None):
+        if id is not None:
```

Comment:
I have suddenly realised what you were concerned about with regard to calling `prep` on an article, and it's because you're doing the crosswalk onto an existing article.

I think that's a high risk way to approach this, instead I would:

1. Take the data from the article form
2. Crosswalk it into a brand new article object with none of the old properties
3. Pass it to the BLL's `create_article` method, with an instruction to detect and merge duplicates

That way this will behave in exactly the same way as a publisher updating an article with a new XML upload behaves.